test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -55,16 +56,6 @@ public function testGetVersionWithValidVersion(string $versionString) : void
         self::assertRegExp('/([0-9]*)(\\.[1-9][0-9]*)*/', $version->getVersion());
     }
 
-    /**
-     * @dataProvider validVersionStringProvider
-     *
-     * @param string $versionString
-     */
-    public function testVersionNumbersAreNormalized(string $versionString) : void
-    {
-        self::assertNotRegExp('/(\\.[0]+)+$/', Version::fromString($versionString)->getVersion());
```

Comment:
not need here because we do not do trailing strip on `Version::fromString()` creation

Reply:
This should be reverted, as explained above. Making versions equivalent at instantiation simplifies reasoning around them later on